lib/istoredp.js

```diff
@@ -60,33 +60,45 @@ const db2Call = (callback,xlib,xdatabase,xuser,xpassword,xipc,xctl,xml_input_scr
         [xml_input_script, db.SQL_PARAM_INPUT, 0],
         [xmlOut, db.SQL_PARAM_OUTPUT, 0],
       ];
+
+    const cleanup = () => {
+      stmt.close();
+      conn.disconn();
+      conn.close();
+    };
         
     if(sync == true) {  // Sync Mode
       stmt.prepareSync(sql);
       stmt.bindParamSync(bindParams);

```

Comment:
We should pass a callback to [bindParamSync](https://github.com/IBM/nodejs-idb-connector/tree/master/docs#dbstmtbindparamsyncparams--callback-deprecated) and check for an error. Then run `cleanup()` if an error occurred.  Otherwise the connection and statement won't be cleaned up properly.